include/vapor/SliceGridAlongPlane.h

```diff
@@ -8,10 +8,13 @@
 class RegularGrid;
 
 struct planeDescription {
+    size_t sideSize;
     std::vector<double> origin;
     std::vector<double> rotation;
     VAPoR::CoordType boxMin;
     VAPoR::CoordType boxMax;
+    VAPoR::CoordType domainMin;
+    VAPoR::CoordType domainMax;
```

Comment:
The 3D `Grid` passed into `SliceGridAlongPlane` already has the function `Grid::GetUserExtents` which returns the data extents. Could that just be used rather than passing around this parameter here?

Reply:
No, and this is reason why #2968 happened.

The SliceRenderer requests a 3D grid based on the SliceRenderer's Box.  When a new Grid is created, its extents can differ from the Box extents that it is fed because some nodes get clipped off.

The fix here is to use a set of domain and box extents that are not derived from the passed 3D grid.

Reply:
Got it